addons/website/views/snippets/s_faq_horizontal.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_faq_horizontal" name="Topics">
+    <section class="s_faq_horizontal" data-oe-shape-data="{'shape':'web_editor/Origins/09_001','flip':[],'showOnMobile':false,'shapeAnimationSpeed':'0'}">
+        <div class="o_we_shape o_web_editor_Origins_09_001"/>
+        <div class="container">
+            <div class="row">
+                <div class="col-12 col-lg-6 offset-lg-3 pt48 pb48">
+                    <h2 style="text-align:center;">Topic Walkthrough</h2>
+                    <p style="text-align:center;">Learn how to quickly set up and start using our services with our step-by-step onboarding process.</p>
+                </div>
+            </div>
+            <div class="row s_nb_column_fixed">
+                <div data-name="Topic" class="s_faq_horizontal_entry col-12">
```

Comment:
I don't think `col-12` is necessary.
Same for the others below 🙂

Reply:
Replaced with `col-lg-12` 👍 
A `row > col-` structure it's needed to trigger the clone/remove features + other useful options like padding, background, visibility etc...